packages/core/src/modules/dif-presentation-exchange/DifPresentationExchangeService.ts

```diff
@@ -67,10 +67,27 @@ export class DifPresentationExchangeService {
 
   public async getCredentialsForRequest(
     agentContext: AgentContext,
-    presentationDefinition: DifPresentationExchangeDefinition
+    presentationDefinition: DifPresentationExchangeDefinition,
+    options?: {
+      filterByNonRevocationRequirements: boolean
```

Comment:
I know we used this parameter before, but it's hard to understand the property name for me.

// defaults to false
"ignoreRevocationStatus": true